tests/functional/adapter/test_timestamps.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from dbt.tests.adapter.utils import test_timestamps
+
+_MODEL_CURRENT_TIMESTAMP = """
+SELECT {{current_timestamp()}} as current_timestamp,
+       {{current_timestamp_in_utc_backcompat()}} as current_timestamp_in_utc_backcompat,
+       {{current_timestamp_backcompat()}} as current_timestamp_backcompat
+"""
+
+class TestCurrentTimestampSnowflake(test_timestamps.TestCurrentTimestamps):
```

Comment:
The pattern I've seen in our code is like [this](https://github.com/dbt-labs/dbt-snowflake/blob/c69e2133769c7f0a9ee30a8e796b4c4bdbd8b7f9/tests/functional/adapter/test_concurrency.py#L9-L14):
```
from dbt.tests.adapter.utils.test_timestamps import BaseCurrentTimestamps

class TestCurrentTimestampSnowflake(BaseCurrentTimestamps):
```

We can keep the discussion about this over [here](https://github.com/dbt-labs/dbt-core/pull/5838/files#r983640640) and [here](https://github.com/dbt-labs/dbt-core/pull/5838/files#r984024424) though. Just pointing this out for sake of completeness.

Reply:
updated